rfd/0004/README.mdx

```diff
@@ -0,0 +1,144 @@
+---
+authors: George Leung <phisgr@gmail.com>
+title: Flows Input/Output Topics
+state: discussion
+---
+
+## Objective
```

Comment:
Perhaps this proposal is missing a WHY?  Why is this important - other than the potential to unify flows?